iTwin/appui
# Visual testing for components

## Changes
Added visual testing for all component examples.


## Testing
To test:
1. Follow [README.md](https://github.com/iTwin/appui/blob/master/full-stack-tests/ui/README.md) instructions to run iTwin.js UI End-To-End Tests
2. Make changes to components and run the tests to make sure that the tests for the components you changed fail. When viewing a failed test's results, you should be able to see the expected, actual, and diff snapshots.
